addons/binding/org.openhab.binding.nibeheatpump/utils/F1X45_utf8.csv

```diff
@@ -0,0 +1,736 @@
+ModbusManager 1.0.8

```

Comment:
This is configuration file from Nibe where all other files are generated via scripts. Removed